AllReadyApp/Web-App/AllReady/Controllers/CampaignController.cs

```diff
@@ -53,19 +52,22 @@ public IActionResult LocationMap(int id)
         public IEnumerable<CampaignViewModel> Get()
         {
             return _dataAccess.Campaigns
+                .Where(c => c.Locked == false)
                 .Select(x => new CampaignViewModel(x));
         }
 
         // GET api/values/5
         [HttpGet("{id}")]
-        public CampaignViewModel Get(int id)
+        public ActionResult Get(int id)

```

Comment:
This is something I noticed when testing the API. Previously there was a HttpNotFound(); line which seems to do nothing since it wasn't being actually returned. However to return it the controller needs to be an action result rather than just the model. I may have misunderstood something here about how the MVC Core controllers work for API responses though?

If this is truly an issue we may need to check the other API endpoints as well.